General.ruleset

```diff
@@ -0,0 +1,115 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<RuleSet Name="General C# rules" ToolsVersion="16.0">
```

Comment:
https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1116.md and https://rules.sonarsource.com/csharp/RSPEC-1199 are the same. Which one do you recommend?

Reply:
They are different. Check their descriptions one more time.

Reply:
Hmmm indeed, these are completely different. I've mixed up something, nevermind.